docs/kyma/05-02-custom-component-installation.md

```diff
@@ -66,12 +66,12 @@ components:
 
 ### Post-installation changes
 
-You can only add a new component after the installation. Removal of the installed components is not possible. To add a component that was not installed with Kyma by default, modify the Installation custom resource.
+You can only add a new component after the installation. Removal of the installed components is not possible. To add a component that was not installed with Kyma by default, perform the following steps.
 
-1. Edit the resource:
+1. Download the current [Installation custom resource](#custom-resource-installation) on the cluster:
```

Comment:
```suggestion
1. Download the current [Installation custom resource](#custom-resource-installation) to the cluster:
```

Reply:
Same as above, I think this should be `from` the cluster.